web/src/app/users/UserContactMethodVerificationForm.js

```diff
@@ -52,7 +52,7 @@ export default function UserContactMethodVerificationForm(props) {
   // We only want to display an error in response to a user action.
   useEffect(() => {
     sendCode().catch(() => {})
-  }, [])
+  }, [sendCode])
```

Comment:
Won't this cause it to trigger every render (new function from `useMutation`)? We only want it to trigger once on mount